luzer/tests/test_options.lua

```diff
@@ -1,10 +1,15 @@
 local luzer = require("luzer")
 
 local args = {
-    max_len = 1024,
-    print_pcs = 1,
-    corpus = "undefined",
-    max_total_time = 60,
-    print_final_stats = 1,
+    "-max_len=1024",
+    "-print_pcs=1",
+    "-max_total_time=60",
+    "-print_final_stats=1",
+    "./corpus/"
```

Comment:
Before the patch `corpus` was `"undefined"`. Why did you change it?

Reply:
returned to original state